### DISCUSSION TITLE
Skirt setting addition to be more efficient with filament

### DISCUSSION THREAD

**Discussion by jermacalocas** [supermerill/SuperSlicer#2830] [Q&A]
did some searching but cant find an answer.

Is there a way to set the skirt to limit by distance extruded instead of loops. Printing on a 500x500 bed I get prints that take up 100x100 and the full bed. I am hoping for a way to set a distance in mm so that it generated just that no matter how many loops it takes or to a certain loop count max. this way no matter how much space is being used on the bed its a more uniform and low amount of waste. 

Unless I'm missing something in plain sight?

**Comment by neophyl** [comment]
Well there is an Expert level setting under Print Settings>Skirt&Brim>Skirt called Minimal filament extrusion length that can be set in mm.  However I think that one is to make sure you are extruding enough filament when the skirt loop is very short.
Normally you would set 1 skirt loop and then a minimum value.  That way it always does 1 loop but on small things it can do more to get the minimum length you set.  

What I don't think it allows is half a loop if its met the minimum mm value.  Although I haven't tested that.

**Reply by jermacalocas** [reply]
yes, i think you get where im going. I also appreciate your view of the minimum mm value set. Ill have to test that and see if it does what im hoping. Even 1 loop on a large print isnt as bad as setting 2-4 because your small prints benefit from that. But where im hoping it doesnt backfire is if the minimum forces like 5+ loops on a small print. 

my hope is to get my settings as universal as possible with the settings that are less important on a print to print basis.
